packages/react-app/src/components/pages/Bounties/Filters/index.tsx

```diff
@@ -58,11 +61,11 @@ const MinMaxFilter = ({ name }: { name?: string }): JSX.Element => (
 )
 
 const HelpLinks = (): JSX.Element => (
-  <HStack>
-    <ColorModeButton>Give us Feedback</ColorModeButton>
-    <ColorModeButton>? Get Help</ColorModeButton>
+  <HStack>  
+    <ColorModeButton url={discordSupportChannelUrl}>Give us Feedback</ColorModeButton>
```

Comment:
Instead of adding URL as an option to `ColorModeButton` can we wrap `AccessibleLink` around it instead?

Here is an example of `AccessibleLink` being used on a `Button`. 

[https://github.com/BanklessDAO/bounty-board/blob/develop/packages/react-app/src/components/pages/Bounties/Bounty/index.tsx#L128](https://github.com/BanklessDAO/bounty-board/blob/develop/packages/react-app/src/components/pages/Bounties/Bounty/index.tsx#L128)

Reply:
Agreed, thanks for the comment and suggestion! Addressed by my most recent push.